### DISCUSSION TITLE
placeholder使用不了

### DISCUSSION THREAD

**Discussion by Ai-Elaina** [walinejs/waline#862] [Q&A] [answered]
为什么，hexo主题为butterfly的placeholder不好使，这是我的配置
![}(CZHA$ UL6T2DA}3GWH RD](https://user-images.githubusercontent.com/63330802/158100566-56864aec-c063-434c-8a9c-202019cad029.png)

**Comment by lizheming** [comment, marked as answer]
https://github.com/jerryc127/hexo-theme-butterfly/blob/dev/_config.yml#L315 butterfly 的配置在 option 里面，有问题直接咨询下主题作者吧。

**Reply by Ai-Elaina** [reply]
好的谢谢
